FestivalPlanner/films/views.py

```diff
@@ -235,15 +235,16 @@ def get_queryset(self):
                 )
 
         # Read the descriptions.
-        initialize_log(session, action='Read descriptions')
-        film_info_file = self.festival.filminfo_file
+        if not get_log(session):
+            initialize_log(session, action='Read descriptions')
```

Comment:
Please consider to only execute this when the log is written to.

Reply:
I tried an old festival (no descriptions) but it doesn't look bad the way it is.

Not doing it.